### ISSUE TITLE
TPMS app not decoding Toyota Prius Prine sensors

### ISSUE THREAD

**Issue by ifuchs** [sharebrained/portapack-hackrf#191]
I am trying to read my Prius prime 2022 TPMS sensors without any success. I can see that there is a transmission when I use the audio or the capture modes but the TPMS app doesn’t show anything. I have tested with an RTL_SDR (rtl_433 – f 315M – m level) and it works fine, decoding the TPMS and showing the pressure. and temperature properly.

**Comment by helmarw** [comment]
please check here for latest info, could be the same issue but its not solved yet
https://github.com/eried/portapack-mayhem/issues/879

**Comment by ifuchs** [comment]
Yes, that does appear to be the same problem. I guess RTL_433 can deal with these sensors while the Portapack app cannot.
